app/models/parking_spot.rb

```diff
@@ -0,0 +1,5 @@
+class ParkingSpot < ApplicationRecord
+  validates :spot_id, presence: true  
+  validates :spot_floor, presence: true
+  validates :spot_description, presence: true
```

Comment:
why not [validates_presence_of](https://apidock.com/rails/ActiveRecord/Validations/ClassMethods/validates_presence_of)?

```ruby
validates_presence_of(:spot, :spot_floor, :spot_description) // you can validate entity, not only field (:spot over :spot_id)

Reply:
:+1: